architecture/spmc.rst

```diff
@@ -0,0 +1,267 @@
+====
+SPMC
+====
+
+This document describes the SPMC (S-EL1) implementation for OP-TEE.
+More information on the SPMC can be found in the FF-A specification can be
```

Comment:
remove 2nd occurrence of "can be found in the":
```patch
-More information on the SPMC can be found in the FF-A specification can be
-found int the
+More information on the SPMC can be found in the FF-A specification:
 `FF-A spec <https://developer.arm.com/documentation/den0077/latest>`_.
```

Reply:
Done